hybridq/noise/channel/channel.py

```diff
@@ -418,7 +418,7 @@ def _get_position(k):
     # Get matrices
     Matrices = [
         kron(*m)
-        for m in product(*([[Gate(g).Matrix for g in 'IXYZ']] * n_qubits))
+        for m in product(*([[Gate(g, n_qubits=1).Matrix for g in 'IXYZ']] * n_qubits))
```

Comment:
I included `n_qubits=1` here since otherwise the Identity gate causes the default number ofqubits warning when this is called.